k8s/prod/common/financial-remedy/finrem-ns.yaml

```diff
@@ -0,0 +1,40 @@
+---
```

Comment:
this file should be called namespace.yaml
https://github.com/hmcts/cnp-flux-config/pull/671/files#diff-5daf978e60e03b3975b485d5f1b449dcR61